detection_engine.py

```diff
@@ -52,10 +53,11 @@
 is_test = args.is_test
 future_bars = args.future_bars
 volatility_filter = args.volatility_filter
+output_format = args.output_format
```

Comment:
I would recommend handling cases where the user passes in the argument with different casing, e.g. `JSON` vs `json`. The most simple way (read: Least amount of code changes) would be to `output_format = args.output_format.upper()`

Reply:
Fair point, I can make that update.

Reply:
Updated